.github/workflows/podPublish.yml

```diff
@@ -2,9 +2,10 @@ name: Pod-Publish
 on:
   push:
     branches:
-      - main
-      - main_0.1
-      - main_0.2
+      # Push events to branches matching refs/heads/
+      - 'main'
```

Comment:
Just so I understand the goal of these changes:

Is there any real difference between listing the branch names in single quotes or this is just a 'no-op' test to force a merge?

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags 
but i think it will filter out with origin main vs actual ref/heads/main